apps/services/sessions/src/app/sessions/sessions.controller.spec.ts

```diff
@@ -61,16 +281,103 @@ describe('SessionsController', () => {
       // Act
       const res = await server
         .post(`/v1/me/sessions`)
-        .send(createRandomSession(faker.datatype.string(10)))
+        .send(createSessionDto({ actorNationalId: faker.datatype.string(10) }))
 
       // Assert
       expect(res.status).toEqual(403)
     })
   })
 
+  describe('with personal delegation', () => {
+    let app: TestApp
+    let server: request.SuperTest<request.Test>
+    const user = createCurrentUser({
+      scope: [ApiScope.internal, SessionsScope.sessionsWrite],
```

Comment:
Personal delegation should never have the `internal` scope. So this is probably a unnecessary test case and with the `if` statement in the service being unnecessary as well.

Reply:
Removed.